cmd/health-agent/probers.go

```diff
@@ -15,6 +16,11 @@ import (
 
 func setupProbers() (*proberlist.ProberList, error) {
 	pl := proberlist.New("/probers")
+	go func() {
+		// New has to be executed in the goroutine as it could take a long time
+		p := aws.New()
+		pl.Add(p, "/sys/aws", 0)
+	}()
```

Comment:
Please do this in a single line, and change the path:
  go pl.Add(aws.New(), "/sys/cloud/aws", 0)